src/main/java/app/controllers/HomeController.java

```diff
@@ -0,0 +1,12 @@
+package app.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class HomeController {
+    @GetMapping("/hello")
```

Comment:
should the home route just be /  ?

Reply:
Yep, this will change in follow up. This PR is more about actually serving something, so I want to be explicit that it's literally a hello world page.

Reply:
ok makes sense